core/build.gradle

```diff
@@ -25,33 +27,36 @@ configurations.configureEach {
 }
 
 dependencies {
-    implementation project(':grpc-read-db')
+    implementation project(":grpc-read-db")
 
-    implementation 'com.exactpro.th2:common:5.2.0-dev'
-    implementation 'org.slf4j:slf4j-api'
+    api platform("com.exactpro.th2:bom:4.5.0")
+    implementation "com.exactpro.th2:common:5.4.0-dev"
+    implementation "com.exactpro.th2:common-utils:2.2.0-dev"
+    implementation "org.slf4j:slf4j-api"
 
-    implementation('org.apache.commons:commons-dbcp2:2.9.0') {
-        because('connection pool')
+    implementation("org.apache.commons:commons-dbcp2:2.9.0") {
+        because("connection pool")
     }
-    implementation 'org.apache.commons:commons-text:1.9'
-    implementation('com.opencsv:opencsv:5.6') {
-        because('publishes raw messages in csv format')
+    implementation "org.apache.commons:commons-text"
+    implementation("com.opencsv:opencsv:5.8") {
+        because("publishes raw messages in csv format")
     }
 
     implementation "org.jetbrains.kotlinx:kotlinx-coroutines-core:$coroutines_version"
-    implementation 'io.github.microutils:kotlin-logging:3.0.0'
+    implementation "io.github.microutils:kotlin-logging:3.0.5"
+    implementation "com.fasterxml.jackson.core:jackson-databind"
 
-    testImplementation 'org.junit.jupiter:junit-jupiter:5.9.0'
+    testImplementation "org.junit.jupiter:junit-jupiter:5.10.0"
     testImplementation "org.jetbrains.kotlinx:kotlinx-coroutines-test:$coroutines_version"
     testImplementation "com.nhaarman.mockitokotlin2:mockito-kotlin:2.2.0"
     testImplementation "io.strikt:strikt-core:0.34.1"
```

Comment:
I think we need to update this dependency because we have updated kotlin verison